Source/WebCore/css/CSSProperties.json

```diff
@@ -1302,18 +1302,17 @@
         },
         "zoom": {
             "values": [
-                "normal",
-                "reset",
-                "document"
+                "normal"
             ],
             "codegen-properties": {
+                "initial": "initialZoom",
```

Comment:
initialZoom can probably be omitted, it's the default.

Reply:
Removed in https://github.com/WebKit/WebKit/commit/d7bdf9c019c208a0662fbc8579be1653ec346338.